web/static/js/components/questionnaires/QuestionnaireEditor.jsx

```diff
@@ -87,6 +88,21 @@ class QuestionnaireEditor extends Component {
     }
   }
 
+  downloadCsv(e) {
```

Comment:
This is done. This code runs when the user clicks the Download link, applies the `csvForTranslation` function on the current questionnaire and writes the content to an URI encoded binary that the downloads. I intended to refactor this before wrapping up the issue, although the code is not terrible.